src/components/AddBook/AddBook.js

```diff
@@ -18,35 +19,46 @@ const BookAdd = () => {
     return <p>Error occurred while fetching books</p>;
   }
   return (
-    <div className="book-wrap">
+    <div className="book-container">
       {books.map((book) => (
         <div className="books" key={book.item_id}>
-          <ul>
-            <li>
-              {book.category}
-              {' '}
-              <br />
-              {' '}
-              {book.title}
-              {' '}
-              By
-              {' '}
-              {book.author}
-              {' '}
-            </li>
-            <button
-              type="button"
-              className="remove"
-              onClick={() => {
-                dispatch(removeBook(book.item_id));
-              }}
-            >
-              Remove
-            </button>
+          <div className="book-add">
+            <h2 className="category">{book.category}</h2>
+            <h1 className="title">{book.title}</h1>
+            <h3 className="author">{book.author}</h3>
```

Comment:
- Kindly make on your page there is only one h1,  as when you add several book you're gonna have many of them. 
- Kindly make sure to use the right font for the book title as per the provided design:
Your design:
![Screenshot from 2023-04-22 19-49-46](https://user-images.githubusercontent.com/97834160/233799259-83dbbaa5-9505-4c05-afae-05bf1496f556.png)
Zeplin:
![Screenshot from 2023-04-22 19-50-25](https://user-images.githubusercontent.com/97834160/233799299-95bf886e-8e05-4cc1-828d-eed1f2f3da78.png)